lib/rules/selector-pseudo-class-no-unknown/__tests__/index.js

```diff
@@ -28,6 +28,9 @@ testRule({
 		{
 			code: 'a::before { }',
 		},
+		{
+			code: 'a:modal { }',
```

Comment:
Bit of a nit, but it seems like from documentation ([MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/:modal), [CSS spec](https://www.w3.org/TR/selectors-4/#selectordef-modal)) that no attached selector is more idiomatic:

```suggestion
			code: ':modal { }',
```